coffee-backend/src/main/resources/application.properties

```diff
@@ -1,3 +0,0 @@
-server.port=8080
```

Comment:
Add these to the `application.yaml` if you want to use yaml instead ;)

Reply:
This is deleted isn' it?